supergraph.config.js

```diff
@@ -0,0 +1,27 @@
+// PM2 config
+module.exports = {
+    apps : [{
+      name   : "subgraph inventory",
+      script : "index.js",
+      cwd: "./subgraphs/inventory",
```

Comment:
unsure whether those relative paths will work for PM2 -> guess we'll know once it is published